questionnaires/esm28q/esm28q_armt.json

```diff
@@ -0,0 +1,1450 @@
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "1 Not at all "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 Very much"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "esm4_v2",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "esm_location_v2",
+        "form_name": "esm28q",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "Where am I?",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "Home "
+            },
+            {
+                "code": "1",
+                "label": "Family or friend's house "
+            },
+            {
+                "code": "2",
+                "label": "Workplace "
+            },
+            {
+                "code": "3",
+                "label": "Transport "
+            },
+            {
+                "code": "4",
+                "label": "Healthcare facility "
+            },
+            {
+                "code": "5",
+                "label": "Somewhere else indoors "
+            },
+            {
+                "code": "6",
+                "label": "Somewhere else outdoors"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "esm_social_interact_v2",
+        "form_name": "esm28q",
+        "section_header": "",
+        "field_type": "checkbox",
+        "field_label": "Who am I with?",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "On my own "
+            },
+            {
```

Comment:
This needs to be checked

Reply:
After convo with Aki, This is done to ensure that the value of the answers can be consistent with the previous version of the ESM.